yt_dlp/extractor/theplatform.py

```diff
@@ -103,6 +105,36 @@ def _add_chapter(start_time, end_time):
             for chapter in tp_chapters[:-1]:
                 _add_chapter(chapter.get('startTime'), chapter.get('endTime'))
             _add_chapter(tp_chapters[-1].get('startTime'), tp_chapters[-1].get('endTime') or duration)
+        info_keywords_str = info.get('keywords', {str_or_none})
+        tags = []
+        if (info_keywords_str is not None) and (info_keywords_str != ''):
+            tags = re.split(', |; |,', info_keywords_str)
+        location = None
+        series = None
+        season_number = None
+        # The following can be uncommented as soon as #7838 is merged:
+        # media_type = None
+        categories = []
+        categories_data = info.get('categories') or []
+        for x in categories_data:
+            if x.get('name') is not None:
+                # Sometimes, there will be several kinds of categories
+                # in this case, it will have a label field with a value 'category'
+                if (x.get('label') is None) or (x.get('label') == 'category'):
+                    categories.append(x.get('name'))
+
+        # A number of sites have a prefix in front of some info keys followed by a '$' symbol.
+        # Search for known keys with the prefix.
+        for key in info.keys():
+            if re.match('.*\\$region', key):
+                location = info[key]
+            if re.match('.*\\$show', key):
+                series = info[key]
+            if re.match('.*\\$seasonNumber', key):
+                season_number = str_to_int(info[key])
+            # the following can be uncommented as soon as #7838 is merged:
+            # if (re.match('.*\programmingType', key)) or (re.match('.*\type', key)):
+                # media_type = info[key]
```

Comment:
#7838 was merged already without adding the `media_type` field -- feel free to propose its addition in this PR

Reply:
I've added it - see the description and my comment below.